common/src/main/java/com/pg85/otg/generator/resource/ReedGen.java

```diff
@@ -99,6 +100,11 @@ public void spawn(LocalWorld world, Random rand, boolean villageInChunk, int x,
         {
             return;
         }
+        
+        if (!bLoaded  ) {
```

Comment:
Could put this in a method in Resource instead, no repeating code for each Gen class?